### DISCUSSION TITLE
Gogs 0.12.10 has been released!

### DISCUSSION THREAD

**Discussion by unknwon** [gogs/gogs#7120] [Announcements]
Hello everyone,

Hope you have pleasant experiences with Gogs so far, and we’re excited to announce that Gogs 0.12.10 is released!

You can find binary downloads on our [website](https://gogs.io/docs/installation/install_from_binary) and the [GitHub release](https://github.com/gogs/gogs/releases/tag/v0.12.10) page.

The CHANGELOG for this release is also available on the [GitHub release](https://github.com/gogs/gogs/releases/tag/v0.12.10) page.

Please enjoy and happy coding!

Please comment on this thread if you spot any packaging issues for the release. For general feature requests and bug reports, please use our [curated issue forms](https://github.com/gogs/gogs/issues/new/choose) :)

### Backwards compatibility

You may upgrade your installation directly to this version if you're running Gogs 0.8.5 or newer.

**Comment by wandersonwhcr** [comment]
ty @unknwon
